internal/config/config.go

```diff
@@ -10,6 +10,8 @@ import (
 const (
 	DefaultFileName = "nri-kubernetes"
 	DefaultFilePath = "/etc/newrelic-infra"
+	DefaultTimeout  = 5000 * time.Millisecond
```

Comment:
isn't that short? I would bump it to 10seconds.
Since after a connection error we often restart the pod having a long timeout should not be a bit drawback

Reply:
Yes, I can put 10s, this is the default we had.

Reply:
done